cmd/scollector/collectors/keepalived_linux.go

```diff
@@ -0,0 +1,109 @@
+package collectors
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/mjibson/snmp"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type vrrpInstanceEntry struct {
```

Comment:
You must listen to this song before committing: http://mirrors.nycbug.org/pub/OpenBSD/songs/song35.mp3